### PR TITLE
Fix builder return type

### PR DESCRIPTION
**Pull request by sven1103**
Many thanks for contributing to this project!

**PR Checklist**
Please fill in the appropriate checklist below (delete whatever is not relevant). These are the most common things requested on pull requests (PRs).

- [ ] This comment contains a description of changes (with reason)
- [ ] Referenced issue is linked
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] Documentation in `docs` is updated
- [ ] `CHANGELOG.rst` is updated

**Description of changes**
Please state what you've changed and how it might affect the user.

**Technical details**
Please state any technical details such as limitations, reasons for additional dependencies, benchmarks etc. here.

**Additional context**
Add any other context or screenshots here.

### CODE REVIEW

File: src/main/groovy/life/qbic/datamodel/dtos/portal/PortalUser.groovy [hunk 1]
```diff
@@ -30,7 +30,7 @@ class PortalUser extends Person {
         }
 
         @Override
-        Person build() {
+        PortalUser build() {
```
Comment (KochTobi) [inline src/main/groovy/life/qbic/datamodel/dtos/portal/PortalUser.groovy:33]: Are there tests missing? Or why was it not detected in the first place?

**Review by KochTobi — CHANGES_REQUESTED** [review]
Please add respective tests as well

**Review by KochTobi — APPROVED** [review]

**Comment by KochTobi** [comment]
The code looks good. Does that also need to go into the changelog?

**Comment by sven1103** [comment]
> The code looks good. Does that also need to go into the changelog?

dont think so, as it is a fix of a not released feature

**Pull request merged**